ref/requirements.md

```diff
@@ -77,7 +77,7 @@ Anbox Cloud currently supports the following LXD versions:
 <a name="juju-version"></a>
 ### Juju version
 
-Anbox Cloud requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. Starting with Anbox Cloud 1.13, the required Juju version is **2.9**. Earlier versions require Juju **2.8**.
+The regular Anbox Cloud variant requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. Starting with Anbox Cloud 1.13, the required Juju version is **2.9**. Earlier versions require Juju **2.8**.
```

Comment:
```suggestion
The regular Anbox Cloud variant requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. The minimum required Juju version is **2.9**.
```

Lets drop the mention of 2.8 entirely. We support/can work with 2.9+. CI actually tests 3.1 today by default.